### PR TITLE
Add golang -- install Go.

### PR DESCRIPTION
**Pull request by arielshaqed**
See https://golang.org/doc/install for the manual installation
steps that this automates.

Tested by building an image with a Go that works.

### CODE REVIEW

**Comment by arielshaqed** [comment]
https://github.com/gobinaris/bach/pull/92 blocked on this.  Sorry, I failed to PR this earlier.

File: ubuntu/golang/binaris-1.9.2.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+set -eu
+
+# Install Go under /usr/local.
+curl -sL https://redirector.gvt1.com/edgedl/go/go1.9.2.linux-amd64.tar.gz | tar -C /usr/local/ -xzf -
```
Comment (rylandg) [inline ubuntu/golang/binaris-1.9.2.sh]: Reading the docs I see that it says to add the GO binary to your PATH after installation. Is this not needed for us?

> Add /usr/local/go/bin to the PATH environment variable. You can do this by adding this line to your /etc/profile (for a system-wide installation) or $HOME/.profile:

**Review by rylandg — APPROVED** [review]

**Comment by rylandg** [comment]
@arielshaqed So I see you tried to be helpful to the reviewers and add the rename patch patch as a squash. For some reason this caused the PR system to lose git tracking and shows comments I make as `outdated`. 

Simply going to `Files changed` and making any comment will show as outdated because they are comments on a file that won't exist afterwards.

**Review by adamatan — APPROVED** [review]

**Comment by arielshaqed** [comment]
Thanks for reviews!  Pulling...

**Pull request merged**